src/makeCalculator.js

```diff
@@ -37,7 +37,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (number) => {
+      result += number;
+
+      return calculator;
+    },
+    subtract: (number) => {
+      result -= number;
+
+      return calculator;
+    },
+    multiply: (number) => {
+      result *= number;
+
+      return calculator;
+    },
+    divide: (number) => {
+      if (result === 0) {
+        return 'You cannot divide by zero';
```

Comment:
Instead of returning a string when dividing by zero, it's better to throw an error to make it explicit that something went wrong.